charts/kubecost-cloud-agent/values.yaml

```diff
@@ -1,4 +1,33 @@
+# Kubecost running as an Agent is designed for external hosting. The current setup deploys a
+# kubecost-agent pod and prometheus server. 
+
+# Sets the image version for the Kubecost agent.
+imageVersion: lunar-sandwich.v0.1.2
+
+# Sets the destination where the Kubecost agent connects.
+cloudReportingServer: collector.app.kubecost.com:31357
+
+# Cloud Agent enables specific features designed to enhance the metrics exporter deployment
+# with enhancements designed for Kubecost Cloud 
+cloudAgent: true 
+cloudAgentKey: ""
+cloudAgentClusterId: my-cluster
```

Comment:
This is the only default I think is potentially problematic, as we want to required this be set externally. Is the best practice here to have the chart error if the variable is unset?

Reply:
Yes, if this is a required value and must be dynamic in nature, the value should be set to empty string in the parent values file with a validation check to ensure it is not empty upon deployment.